tools/autograd/load_derivatives.py

```diff
@@ -253,14 +260,12 @@ def repl(m: Any) -> str:
     return updated_derivatives
 
 def is_forward_derivative_definition(all_arg_names: List[str], names: Tuple[str, ...]) -> bool:
-    if len(names) > 1:
-        # Forward definition are always for a single output at a time
-        return False
-    name = names[0]
-    if name not in all_arg_names:
-        return True
-    else:
-        return False
+    for name in names:
+        if name not in all_arg_names:
```

Comment:
Is it possible to have an input and an output with the same name?

Reply:
Yes, the two ops where this happens are `symeig` and `eig`. Neither actually have a formula wrt that argument since it is a boolean. I just added an assert that checks that the name is not both input/output - both are deprecated though so no one should be trying to add forward AD formulas for those outputs.